js/controller.js

```diff
@@ -620,10 +629,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
-    sendDiscoveryClickEvent: function(selectedContentData) {
+    sendDiscoveryClickEvent: function(selectedContentData, isAutoUpNext) {
```

Comment:
All of the calls to this method should be updated to include "false" so we aren't relying on using undefined as false.